README.md

```diff
@@ -179,6 +179,8 @@
 
 - [fitx-widget.js](https://gist.github.com/DanielStefanK/487175b6f65ede401e37ee4848970176) - Workload of a FitX gym.
 
+- [nfl-team-schedule-widget](https://github.com/brianwalborn/nfl-team-schedule-widget) - the current season schedule for your favorite NFL team
```

Comment:
```suggestion
- [nfl-team-schedule-widget](https://github.com/brianwalborn/nfl-team-schedule-widget) - The current season schedule an NFL team.
```

Reply:
Sorry for the delayed feedback. This suggestion should keep the linter happy.

Reply:
No worries! Addressed the missing punctuation.